source/elements/oneTBB/source/info_namespace.rst

```diff
@@ -16,8 +16,13 @@ Interfaces to query information about execution environment.
     namespace oneapi {
     namespace tbb {
         using numa_node_id = /*implementation-defined*/;
+        using core_type_id = /*implementation-defined*/;
+
         namespace info {
             std::vector<numa_node_id> numa_nodes();
+            std::vector<core_type_id> core_types();
+
+            int default_concurrency(task_arena::constraints c);
             int default_concurrency(numa_node_id id = oneapi::tbb::task_arena::automatic);
```

Comment:
As far as I understand, there is no necessity in having `default_concurrency(numa_id)` in presence of a more general `default_concurrency(constraints)` function. Can we remove it from the spec in a similar way as constructor for `constraints` structure below is going to be removed? I.e. `default_concurrency(numa_id)` does not exist in the specification starting from C++20.

Reply:
I don't think that we really should delete this overload. The constructor is removed in case of using C++20 to allow designated initialization. Removing the overload doesn't bring any new abilities to customers, so I think we should leave it as is. @alexey-katranov what do you think about it?

Reply:
It was discussed offline that removing the overload is not worth the legacy interfaces break.